brage-import/src/main/java/no/sikt/nva/brage/migration/NvaType.java

```diff
@@ -5,6 +5,7 @@
 @JacocoGenerated
 public enum NvaType {
     BOOK("Book"),
+    BOOK_OF_ABSTRACTS("Book of abstracts"),
```

Comment:
BookOfAbstracts, whitespace enums for the frontend is a pain to handle.

Reply:
This is temporary value -> this will be mapped to InstanceType: ReportBookOfAbstracts in NVA.